### ISSUE TITLE
Wrong Java classpath set in generic launcher POSIX shellscript for BlueJ 5.2.0

### ISSUE THREAD

**Issue by GrandAdmiralThrawn** [k-pet-group/BlueJ-Greenfoot#2232]
When installing BlueJ 5.2.0 using the generic installer ("BlueJ-generic-520.jar") on a UNIX-like operating system, it creates a small POSIX shellscript named "bluej" for launching the program.

In that file you specify a classpath that was still correct for BlueJ 5.1.0:

`CP="$APPBASE/lib/bluej.jar"`

However, this breaks since BlueJ 5.2.0 with the following error message:

```
Error: Could not find or load main class bluej.Boot
Caused by: java.lang.ClassNotFoundException: bluej.Boot
```

It seems the class was moved from bluej.jar to boot.jar, so the classpath set in the launcher needs to be changed accordingly:

`CP="$APPBASE/lib/boot.jar"`

After the change, BlueJ 5.2.0 works normally, as expected.

**Comment by neilccbrown** [comment]
Thanks for reporting this.  Now fixed, and the generic installer for 5.2.0 on the main page has been replaced with the working one containing the fix you suggested.

**Comment by GrandAdmiralThrawn** [comment]
Confirmed working, thanks for the quick fix!
